src/components/TeachingBubble/TeachingBubble.Props.ts

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { TeachingBubble } from './TeachingBubble';
+
+/**
+ * Checkbox class interface.
+ */
+export interface ITeachingBubble {
+
+}
+
+/**
+ * TeachingBubble component props.
+ */
+export interface ITeachingBubbleProps extends React.Props<TeachingBubble> {
+  /**
+   * The background color of the TeachingBubble
+   * @default white
+   */
+  backgroundColor?: TeachingBubbleBGColor;
+
+  /**
+   * A title for the Teaching Bubble.
+   */
+  title?: string;
+
+  /**
+   * An Image for the Teaching Bubble.
+   */
+  img?: string;
+
+  /**
+   * A body for the Teaching Bubble.
+   */
+  body?: string;
```

Comment:
This is partly where I need to learn how you implement these components in an actual application. For me, if a Teaching Bubble always has a set of 6 elements in it (image, title, body copy, accept button, cancel button), I would typically code those into the props so that they have a super simple interface to the pattern.

If we just pass through children the developer will be required to know what markup to use and the logic of what markup to include based on the various combinations of layouts.

Pushing everything through as props we can create the proper markup, wrapper divs, classes, tags etc.